app/types/transport/requests/task/change-task-status.ts

```diff
@@ -0,0 +1,25 @@
+import { NewMessage } from 'ctproto';
+
+export interface ChangeTaskStatusPayload {
+  /**
+   * Id of the task status of which should be updated
+   */
+  taskId: string
+
+  /**
+   * New task status if exists
+   */
+  newStatusId?: string
```

Comment:
why it is optional? Looks illogical

Reply:
new status is not present in case you move task to `Unsorted` column

Reply:
got it